Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs

```diff
@@ -693,80 +695,23 @@ recentSearchesAndFavourites state push =
   , cornerRadii $ Corners (4.0) true true false false
   ][ savedLocationsView state push
    , recentSearchesView state push
-   , bannerView state push
+   , if(state.props.isBanner) then genderBannerView state push else linearLayout[][]
```

Comment:
@Akash2172  instead of using an empty linear layout, this can be concatenated instead.